src/components/FirstTimeUserFlow/RoommatePrefForm.js

```diff
@@ -0,0 +1,107 @@
+import * as React from 'react';
+import { useState, useEffect } from 'react';
+import Grid from '@mui/material/Grid';
+import Typography from '@mui/material/Typography';
+import TextField from '@mui/material/TextField';
+import {
+	FormControlLabel,
+	FormControl,
+	FormLabel,
+	FormGroup,
+	RadioGroup,
+	Radio,
+	Select,
+	MenuItem,
+	Box,
+	Chip,
+	Checkbox,
+} from '@mui/material/';
+
+const dislikeOptions = ['Smoking', 'Alcohol', 'Pets', 'Couples', 'Children', 'Parties'];
+const hobbies = [
+	'Reading',
+	'Watching TV',
+	'Family Time',
+	'Movies',
+	'Fishing',
+	'Computer',
+	'Gardening',
+	'Renting Movies',
+];
+export default function RoommatePrefForm() {
+	const initDislikeSelection = dislikeOptions.reduce((acc, val) => {
+		acc[val] = false;
+		return acc;
+	}, {});
+	// console.log(dislikesObj);
+	const [bedtime, setBedtime] = useState('');
+	const [dislikes, setDislike] = useState(initDislikeSelection);
+
+	const handleDislikesChange = (event) => {
+		setDislike({
+			...dislikes,
+			[event.target.name]: event.target.checked,
+		});
+	};
+
+	return (
+		<React.Fragment>
+			<Typography variant="h6" gutterBottom>
+				Roommate Preference
+			</Typography>
+			<Grid container spacing={3}>
+				<Grid item xs={12} md={6}>
+					<FormLabel>Desired bedtime</FormLabel>
+					<TextField
+						id="time"
+						type="time"
+						defaultValue="22:00"
+						// className={classes.textField}
```

Comment:
```suggestion
						// className={classes.textField}
```
war on non-comment comments